scripts/linters/pylint_extensions.py

```diff
@@ -1960,6 +2059,427 @@ def process_tokens(self, tokens):
                         line=line_num)
 
 
+class ExceptionalTypesCommentChecker(checkers.BaseChecker):
+    """Custom pylint checker which checks that there is always a comment
+    for exceptional types in the backend type annotations.
+    """
+
+    EXCEPTIONAL_TYPE_STATUS_DICT = {
+        'type_comment_present': False,
+        'type_comment_line_num': 0,
+        'is_function_signature_block': True,
+        'outside_args_section': True,
+        'type_present_inside_arg_section': False,
+        'type_present_inside_return_section': False,
+        'type_present_in_function_signature': False,
+        'args_section_end_line_num': 0,
+        'func_def_start_line': 0,
+    }
+
+    EXCLUDED_DIRS_HAVING_EXCEPTIONAL_TYPE_COMMENTS = copy.deepcopy(
+        EXCLUDED_TYPE_COMMENT_DIRECTORIES
+    )
```

Comment:
This constant shouldn't be changed, so why do you need to deepcopy it?

Reply:
removed